docs/source/reference/rest.md

```diff
@@ -175,10 +175,10 @@ r.raise_for_status()
 r.json()
 ```
 
-The same API token can also authorize access to the [Jupyter Notebook REST API][]
-provided by notebook servers managed by JupyterHub if it has the necessary `access:users:servers` scope:
+The same API token can also authorize access to the [Jupyter Notebook REST API](https://jupyterhub.readthedocs.io/en/stable/reference/rest-api.html#/)
+provided by notebook servers managed by JupyterHub if it has the necessary `access:users:servers` scope.
 
-(api-pagination)=
+<p>&nbsp;</p>
```

Comment:
This is a MyST link target. Please don't remove them. In general, it's a good idea to. not remove things if you don't recognize them. THey are probably there for a reason, and you can ask what they are for.